app/styles/main.scss

```diff
@@ -23,6 +23,17 @@ html, body {
   height: 100%;
 }
 
+.flexbox-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 100vh;
+}
+
+.flexbox-spacer {
+  flex-grow: 2;
```

Comment:
Can you elaborate a little? What kind of designs are you thinking about? And which specific pages?